app/controllers/preference_sets_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PreferenceSetsController < ApplicationController
+  def table_state
+    skip_authorization
```

Comment:
I'll need to think how we want to handle authorization. Pundit normally does it based on current_user + some record (this case the preference set). We can totally include that inside the service (we'll need to refetch the user or pass in the current user, quick change).

Reply:
fair enough lol.